src/garage/replay_buffer/replay_buffer.py

```diff
@@ -12,173 +12,75 @@
 import abc
 from abc import abstractmethod
 
-import numpy as np
-
 
 class ReplayBuffer(metaclass=abc.ABCMeta):
     """Abstract class for Replay Buffer.
 
     Args:
         env_spec (garage.envs.EnvSpec): Environment specification.
-        size_in_transitions (int): total size of transitions in the buffer
-        time_horizon (int): time horizon of rollout.
-
+        capacity_in_transitions (int): capacity of the replay buffer,
```

Comment:
we should keep this terminology consistent in the interfaces

do we have Transitions, TimeSteps, Steps, or Samples? 

I slightly prefer TimeSteps and I think we already have that in dtypes, so this would be `capacity_in_time_steps`. And `SampleBatch` should actually be `TimeStepBatch`

If we thinks that's too wordy, we should look at the entire termnology family.